.github/workflows/update-viablestrict.yml

```diff
@@ -42,4 +45,5 @@ jobs:
         run: |
           git config --global user.email "pytorchmergebot@users.noreply.github.com"
           git config --global user.name "PyTorch MergeBot"
-          echo "::debug::Set the latest sha variable to be ${{ steps.get-latest-commit.outputs.latest_viable_sha }}"
+          echo "Set the latest sha variable to be ${{ steps.get-latest-commit.outputs.latest_viable_sha }}"
```

Comment:
does this work if there's no green commits since the last time it updated?

Reply:
if there are no green commits since the last update, it will return the latest commit on viable/strict